### DISCUSSION TITLE
feature request : quick payment button

### DISCUSSION THREAD

**Discussion by ayoub-hs** [Blair2004/NexoPOS#2091] [Ideas]
can you add quick pay button in pos screen that will submit full cash payment without confirmation popup. and it would be great if we assign to it a keyboard shortcut to speed up sales in grocery shop

**Comment by Blair2004** [comment]
Hi,
Unfortunately this is for many reasons not something we want to do:

- the current process (Pay > Full Payment) is the more safe method
preventing inaccurate click/touch
- that button will overload the cart bottom
- no confirmation will increase the error rate

Thank you for the suggestion however.
